src/server/routes/universal.js

```diff
@@ -4,7 +4,9 @@ import { universal, homepage } from '../handlers/universal';
 
 const router = Router();
 
-router.get('/', homepage);
+if (process.env.NODE_ENV !== 'development') {
```

Comment:
It feels a bit weird to be adding this conditional on this level.
Wouldn't it make more sense to have devel/prod logic inside the `homepage` handler (https://github.com/canonical-websites/build.snapcraft.io/blob/master/src/server/handlers/universal.js#L12-L14)?

Reply:
I originally had in the homepage handler but figured short-circuiting earlier would be better. Happy to move it if you think it would be better in the handler.

Reply:
Yes please, I think this file is good just to list the routes without any logic. We already have some ENV checks inside the handlers, so for consistency lets move it there as well.